gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/linker/ClusterPropertyLinker.java

```diff
@@ -43,4 +45,9 @@ public void link(Bundle filteredBundle, Bundle bundle, File rootFolder) {
         idsToRemove.forEach(clusterPropertyMap::remove);
     }
 
+    @Override
+    public void link(Set<Dependency> dependencies) {
```

Comment:
Can you please explain how does it work?
Why are you changing the type from CW to ENCAS?

Reply:
all cluster wide properties are changed to global environment properties when writing it to yml files
as a result when matching dependency with entities in the bundle its failing to load
So we are changing this in policy dependencies also